core/data/src/test/java/uk/gov/gchq/gaffer/data/util/ElementUtil.java

```diff
@@ -81,10 +81,10 @@ public static void assertElementEquals(final Iterable<? extends ElementId> expec
                 }
             }
 
-            assertTrue("\nMissing entities:\n" + missingEntities.toString()
-                            + "\nUnexpected entities:\n" + incorrectEntities.toString()
-                            + "\nMissing edges:\n" + missingEdges.toString()
-                            + "\nUnexpected edges:\n" + incorrectEdges.toString(),
+            assertTrue("\nMissing entities:\n(" + missingEntities.size() + ") " + missingEntities.toString()
```

Comment:
It's purely a cosmetic thing to make it more readable but this may be better:
```suggestion
            assertTrue("\nMissing " +  missingEntities.size() + " entities:\n" + missingEntities.toString()
```

In the grand scheme of things, it doesn't matter that much

Reply:
I think I'll leave this for now